test/black-box/profile/ProfileRemoveCommand.test.ts

```diff
@@ -68,5 +104,4 @@ describe('ProfileRemoveCommand', () => {
         expect(res.error.error.success).toBe(false);
         expect(res.error.error.message).toBe('Entity with identifier ' + createdProfile2.name + ' does not exist');
```

Comment:
use new Exception().getMessage().
also I've been removing all expect(res.error.error.success).toBe(false); tests whenever I come by one, since the res.expectStatusCode(404); basicly tests the same thing and you get to see the result failing error message test much faster.